### DISCUSSION TITLE
如何向协议端“发送json消息”？

### DISCUSSION THREAD

**Discussion by pilgrim773** [MrXiaoM/Overflow#60] [问题解答]
我看`napcat`那边对于音乐卡片的分享方式要么是`json`消息要么是`CQCode`，已知后者不在本项目支持范围，那么如何发送`json`？我看指南写了已实现 

> - [x] 将 mirai 消息段序列化/反序列化为 json

是否和这个有关系、如何使用呢？

**Comment by MrXiaoM** [comment]
+ 如果说的是 json 卡片消息，用 `LightApp`
+ 如果说的是 json 数组消息段，详见[开发文档](https://mirai.mrxiaom.top/docs/dev/#%E5%90%91-onebot-%E5%8F%91%E9%80%81%E8%87%AA%E5%AE%9A%E4%B9%89-action)

**Reply by pilgrim773** [reply]
好，已经试了两种方法，协议端都有报错。应该要问`napcat`那边的规范到底是怎样。谢谢

**Comment by MrXiaoM** [comment]
README 中的 `将 mirai 消息段序列化/反序列化为 json` 意为，可将 mirai 的消息类型与 onebot 消息段自动地互相转换
